go/vt/vitessdriver/doc.go

```diff
@@ -38,57 +38,9 @@ features like consistent, application-transparent resharding:
 When you scale up the number of shards, vtgate becomes aware of the new shards.
 You do not have to update your application at all.
 
-Another Vitess feature is the vtgate v3 API. Once enabled, a sharded Vitess
-setup becomes a drop-in replacement for an unsharded MySQL/MariaDB setup.
-
-
-Vtgate API Versions
-
-This driver supports the vtgate API versions v1 and v3, but not v2.
-
-We recommend you use the v1 API for initial experiments and then switch to v3
-later because v3 requires an additional setup step.
-
-The API versions differ as follows:
-
-
-Vtgate API v1
-
-v1 requires each query to specify the target shard. vtgate then routes the query
-accordingly.
-
-This driver supports the v1 API at the connection level. For example, to open a
-Vitess database connection for queries targeting shard "0" in the keyspace
-"test_keyspace", call:
-
-  vitessdriver.OpenShard(vtgateAddress, "test_keyspace", "0", 30*time.Second)
-
-This mode is recommended for initial experiments or when migrating from a
-pure MySQL/MariaDB setup to an unsharded Vitess setup. For a sharded Vitess
-setup we recommend to move to the v2 or v3 API because only these APIs are
-agnostic of resharding events.
-
-
-Vtgate API v2
-
-v2 requires each query to specify the affected sharding keys ("keyspace_id").
-
-Based on the provided sharding key, vtgate then routes the query to the shard
-that currently covers that sharding key.
-
-This driver currently does not support the v2 API because it's difficult to
-enhance the Go SQL interface with the sharding key information at the
-query-level.
-Instead, it's recommended to use the v3 API.
-
-
-Vtgate API v3
-
-v3 automatically infers the sharding key (and therefore the target shard)
-from the query itself.
-
-It also maintains secondary indexes across shards and generates values for
-AUTO_INCREMENT columns.
+VTGate is capable of breaking up your query into parts, routing them to the
+appropriate shards and combining the results, thereby giving the semblance
+of a unified database.
 
 See the vtgate v3 Features doc for an overview:
```

Comment:
I would add here a sentence which introduces the term "vtgate v3" before you're using it for the first time e.g.

> Within the Vitess codebase, this sharding transparent API is dubbed "vtgate v3".